chart/hooks/pre-upgrade/remove-deployment-updater-readiness.sh

```diff
@@ -18,8 +18,18 @@ spec:
       - name: cc-deployment-updater-cc-deployment-updater
         readinessProbe: ~
 '
+set +o pipefail
+scheduler_list=$(kubectl get statefulsets --namespace "$NAMESPACE" | grep scheduler | cut -d " " -f 1)
+set -o pipefail
```

Comment:
I would prefer to use the `kubectl` command to specify exactly the output we need instead of post-processing it with grep/awk/sed/cut etc, which can break when the default output format changes. E.g.

```
scheduler_list=$(kubectl get statefulsets \
    --namespace "${NAMESPACE}" \
    --selector quarks.cloudfoundry.org/quarks-statefulset-name=scheduler \
    --no-headers=true \
    --output custom-columns=:metadata.name \
)
```

Reply:
@ShuangMen I just realized that `quarks.cloudfoundry.org//quarks-statefulset-name` might not be the correct selector; I don't have a multi-az cluster to test.

I think `quarks.cloudfoundry.org/instance-group-name` is actually the correct label.

Sorry about the confusion!

Reply:
@jandubois, thanks. I update the code based on your comments.